README
NAME
    Dist::Zilla::PluginBundle::RSRCHBOY - Zilla your distributions like
    RSRCHBOY!

VERSION
    This document describes version 0.052 of
    Dist::Zilla::PluginBundle::RSRCHBOY - released November 12, 2014 as part
    of Dist-Zilla-PluginBundle-RSRCHBOY.

SYNOPSIS
        # in your dist.ini...
        [@RSRCHBOY]

DESCRIPTION
    This is RSRCHBOY's current Dist::Zilla "dist.ini" config for his
    packages. He's still figuring this all out, so it's probably wise to not
    depend on this being too terribly consistent/sane until the version gets
    to 1.

METHODS
  release_plugins
    Plugin configuration for public release.

  author_tests
  meta_provider_plugins
    Plugins that mess about with what goes into META.*.

  configure
    Preps plugin lists / config; see Dist::Zilla::Role::PluginBundle::Easy.

  ensure_current
    Sometimes things change. (I know, I know, the horror!) This seeks to
    minimize that pain by automatically making what changes it can.

  stopwords
    A list of words our POD spell checker should ignore.

OPTIONS
  sign (boolean; default: true)
    On release, use your gpg key to sign the version tag created (if you're
    using git) and also generate a SIGNATURE file.

    See also Dist::Zilla::Plugin::Signature.

  tweet (boolean; default: false)
    If set to a true value, we'll use Dist::Zilla::Plugin::Twitter to tweet
    when a release occurs.

  github (boolean; default: true)
    This enables various GitHub related plugins to update distribution and
    GitHub metadata automatically.

  install_on_release (boolean; default: true)
    After a release, install the distribution locally. Our default install
    command is (from inside the built release directory):

        cpanm .

    You can change this by setting the "InstallRelease.install_command"
    option.

BUNDLED PLUGIN OPTIONS
    It's possible to pass options to our bundled plugins directly:

        ; format is Plugin::Name.option
        [@RSRCHBOY]
        GatherDir.exclude_filename = cpanfile

    For information on specific plugins and their options, you should refer
    to the documentation of Dist::Zilla::Role::PluginBundle::Config::Slicer.

SEE ALSO
    Please see those modules/websites for more information related to this
    module.

    *   Dist::Zilla::Role::PluginBundle::Easy

    *   Dist::Zilla::Role::PluginBundle::PluginRemover

    *   Dist::Zilla::Role::PluginBundle::Config::Slicer

    *   Config::MVP::Slicer

SOURCE
    The development version is on github at
    <http://https://github.com/RsrchBoy/dist-zilla-pluginbundle-rsrchboy>
    and may be cloned from
    <git://https://github.com/RsrchBoy/dist-zilla-pluginbundle-rsrchboy.git>

BUGS
    Please report any bugs or feature requests on the bugtracker website
    https://github.com/RsrchBoy/dist-zilla-pluginbundle-rsrchboy/issues

    When submitting a bug or request, please include a test-file or a patch
    to an existing test-file that illustrates the bug or desired feature.

AUTHOR
    Chris Weyl <cweyl@alumni.drew.edu>

  I'm a material boy in a material world
    Please note I do not expect to be gittip'ed or flattr'ed for this work,
    rather it is simply a very pleasant surprise. I largely create and
    release works like this because I need them or I find it enjoyable;
    however, don't let that stop you if you feel like it ;)

    Flattr this
    <https://flattr.com/submit/auto?user_id=RsrchBoy&url=https%3A%2F%2Fgithu
    b.com%2FRsrchBoy%2Fdist-zilla-pluginbundle-rsrchboy&title=RsrchBoy's%20C
    PAN%20Dist-Zilla-PluginBundle-RSRCHBOY&tags=%22RsrchBoy's%20Dist-Zilla-P
    luginBundle-RSRCHBOY%20in%20the%20CPAN%22>, gittip me
    <https://www.gittip.com/RsrchBoy/>, or indulge my Amazon Wishlist
    <http://bit.ly/rsrchboys-wishlist>... If you so desire.

CONTRIBUTORS
    *   Neil Bowers <neil@bowers.com>

    *   Sergey Romanov <complefor@rambler.ru>

COPYRIGHT AND LICENSE
    This software is Copyright (c) 2013 by Chris Weyl.

    This is free software, licensed under:

      The GNU Lesser General Public License, Version 2.1, February 1999



Changes
Revision history for Dist-Zilla-PluginBundle-RSRCHBOY

0.052     2014-11-12 22:09:14 PST-0800
	* [dzil] EOLTests becomes Test::EOL

0.051     2014-10-14 22:34:15 PDT-0700
	* Add Dist::Zilla::Plugin::Run::AfterMint as a minting prereq.

0.050     2014-07-05 15:25:22 PDT-0700
	* Add missing prepreq to dist.ini

0.049     2014-05-12 12:52:47 PDT-0700
	* [dzil] drop errant File::Slurp usage (closes #6)
	* [dzil] drop InstallGuide, as all my releases are ABENDing
	* [pw]   restructure our Authors section a touch; add HTML images / links

0.048     2014-04-29 14:44:00 PDT-0700
	* [pw] add a custom Authors plugin

0.047     2014-04-17 15:41:19 PDT-0700
	* [dzil] switch from NoTabsTests to Test::NoTabs
	* [pw] use SingleEncoding (thanks! issue #5)
	* [dzil] use Git::Raw to do *some* of the git bits for now

0.046     2014-04-11 14:58:09 PDT-0700
	* [dzil] Add a helper role for Git::Raw operations
	* [pw]   Move Pod::Weaver deps into dist.ini
	* [dzil] move our dependency listing from the bundle to the dist.ini
	* [dzil] add github_user overrides; use GithubMeta plugin

0.045     2014-04-10 17:29:59 PDT-0700
	* [dzil] add CheckSelfDependency (ensures no recursive deps)
	* [dzil] add missing dzil plugin dep to dist.ini *le sigh*
	* [dzil] make our max_target_perl v5.10.1
	* [dzil] mark ContributorsFromGit minimum as 0.010
	* [dzil] move PAUSE::Encrypted dep declaration to dist.ini
	* [dzil] ...and add missing plugin: Travis::ConfigForReleaseBranch
	* [dzil] add CheckSelfDependency

0.044     2014-03-03 11:59:20 PST-0800
	* [dzil] make POD spelling test happy ('dist' makes them sad!)
	* [dzil] Fix 0.012 release date (github #1; thanks @neilbowers!)
	* [dzil] update travis config
	* [dzil] misc cleanups
	* [dzil] add SchwartzRatio
	* [dzil] move the unincorporated prereqs over into dist.ini
	* [dzil] drop ExtraTests, add RunExtraTests and CheckExtraTests
	* [dzil] auto-remove README.pod, if present (was replaced with the
	  markdown version a while back)

0.043     2013-08-13 01:23:32 PDT-0700
	* [dzil] move "missing prereqs" into dist.ini
	* [dzil] only fetch/update the origin remote once
	* [dzil] add Travis::ConfigForReleaseBranch
	* [dzil] add PromptIfStale

0.042     2013-08-04 15:18:09 PDT-0700
	* add Test::CheckDeps
	* start prepping dist.ini to hold our prereqs (...maybe)
	* use Git::Check::Remote to ensure master and release/cpan are in sync before releasing
	* don't add cpanfile to build in dist.ini, as we're doing it in the bundle now
	* default => builder; plus a note to clean up this area.
	* make copy_from_build a private attribute rather than a method
	* add the PluginRemover role!
	* add ensure_current() method, to help cope with bundle changes
	* require a recent version of autodie
	* rejigger our .travis.yml to be...  well, hopefully better
	* copy cpanfile back into our root; for travis and others

0.041     2013-07-20 13:23:43 PDT-0700
	* [dzil] provide a default authority (cpan:RSRCHBOY)
	* various travis tweaks so we build correctly over there

0.040     2013-07-12 23:34:53 CDT-0500
	* drop requirement on perl v5.10+; not needed for this package and was
	  making testing difficult on travis.
	* [dzil] add Authority, Git::Describe plugins
	* [dzil] add Test::MinimumVersion w/a default of 5.008008

0.039     2013-04-22 21:23:23 PDT-0700
	* [dzil] sync git allow dirty files lists (and ensure the new README.mkdn
	  gets committed appropriately!)

0.038     2013-03-29 22:58:45 PDT-0700
	* [dzil] allow dev-style versions (x.xxxxxx_xx)

0.037     2013-03-28 21:58:31 PDT-0700
	* [dzil] remove TravisYML plugin until it's a little more sorted
	* [dzil] Switch GH readme from POD to Markdown to support badges (travis,
	  etc)

0.036     2013-03-17 22:12:51 PDT-0700
	* [dzil] add TravisYML plugin and configuration

0.035     2013-02-23 10:16:16 PST-0800
	* [dzil] rework how/where we add the git plugins to resolve ordering
	  dependency issues (Push vs CommitBuild)

0.034     2013-02-23 09:51:18 PST-0800
	* [dzil] add Git::CommitBuild to track full CPAN releases

0.033     2013-01-09 16:06:50 PST-0800
	* [dzil] expose trial info in Changes version line
	* [dzil] add cpanfile to list of files allowed to be dirty

0.032     2012-12-21 21:26:53 CST6CDT
	* [weaver]: add GeneratedAttributes, LazyAttributes section
	* [dzil]:	add CPANFile
	* [dzil]: 	add Test::Pod::LinkCheck plugin

0.031     2012-11-26 23:27:18 PST8PDT
	* add ContributorsFromGit to dzil bundle

0.030     2012-11-14 21:45:00 PST8PDT
	* Add CONTRIBUTORS section to our weaver configuration

0.029     2012-11-03 23:26:30 PST8PDT
	* switch to using the dist-zilla specific pluginbundle slicer role
	* drop Test::Portability, as the underlying module has been broken for 6
	  years

0.028     2012-11-03 21:55:56 PST8PDT
	* add lazy/required attributes sections from CollectWithInfo (that is, we
	  now have a nice little bit of intro text).
	* Add 'ROLE PARAMETERS' section

0.027     2012-08-26 12:43:33 America/Los_Angeles
	* actually copy LICENSE back to root now *sigh*
	* add 'coderef' to list of stopwords

0.026     2012-07-13 15:06:31 America/Los_Angeles
	* dzp: CheckChangesHasContent
	* allow "sign" as an option (set to 0/falsey to disable tag and release
	  signing)
	* add Config::MVP::Slicer for broader plugin option coverage
	* add "tweet" option
	* initial start of (real) documentation

0.025     2012-06-30 11:05:55 America/Los_Angeles
	* dzp: Git to 1.121770
	* dzp: enable tag signing

0.024     2012-06-20 21:12:42 America/Los_Angeles
	* add dzp Signature
	* force DZ::PB::Git to 1.121770, for the reworked CommitBuild
	* drop weaver section Consumes; not quite optimal yet

0.023     2012-06-18 17:07:19 America/North_Dakota/New_Salem
	* add MetaProvides::Package, MetaNoIndex
	* add weaver section: Consumes

0.022     2012-06-06 01:44:51 America/Los_Angeles
	* allow .gitignore to be dirty during pre-release
	* break "author" tests config out into author_tests() and allow them to be
	  excluded via "rapid_dev = 1"

lib/Pod/Weaver/PluginBundle/RSRCHBOY.pm
#
# This file is part of Dist-Zilla-PluginBundle-RSRCHBOY
#
# This software is Copyright (c) 2013 by Chris Weyl.
#
# This is free software, licensed under:
#
#   The GNU Lesser General Public License, Version 2.1, February 1999
#
package Pod::Weaver::PluginBundle::RSRCHBOY;
our $AUTHORITY = 'cpan:RSRCHBOY';
$Pod::Weaver::PluginBundle::RSRCHBOY::VERSION = '0.052';
# ABSTRACT: Document your modules like RSRCHBOY does

use strict;
use warnings;

use Pod::Weaver::Config::Assembler;

sub _exp { Pod::Weaver::Config::Assembler->expand_package($_[0]) }
sub _exp2 { [ "\@RSRCHBOY/$_[0]", _exp($_[0]), {} ] }

my $vformat = 'This document describes version %v of %m - released %{LLLL dd, yyyy}d as part of %r.';

# this.... needs some work.

sub mvp_bundle_config {
    return (
        [ '@RSRCHBOY/StopWords', _exp('-StopWords'), {} ],
        [ '@RSRCHBOY/CorePrep',  _exp('@CorePrep'),  {} ],
        _exp2('Name'),
        [ '@RSRCHBOY/Version', _exp('Version'), { format      => $vformat  } ],
        [ '@RSRCHBOY/prelude', _exp('Region'),  { region_name => 'prelude' } ],

        [ 'SYNOPSIS',         _exp('Generic'),      {} ],
        [ 'DESCRIPTION',      _exp('Generic'),      {} ],
        [ 'OVERVIEW',         _exp('Generic'),      {} ],

        [ 'ROLE PARAMETERS', _exp('RSRCHBOY::RoleParameters'), {} ],

        [ 'REQUIRED ATTRIBUTES', _exp('RSRCHBOY::RequiredAttributes'), { } ],
        [ 'LAZY ATTRIBUTES',     _exp('RSRCHBOY::LazyAttributes'),     { } ],

        [ 'ATTRIBUTES',       _exp('Collect'), { command => 'attr'            } ],
        [ 'METHODS',          _exp('Collect'), { command => 'method'          } ],
        [ 'REQUIRED METHODS', _exp('Collect'), { command => 'required_method' } ],
        [ 'FUNCTIONS',        _exp('Collect'), { command => 'func'            } ],
        [ 'TYPES',            _exp('Collect'), { command => 'type'            } ],
        [ 'TEST_FUNCTIONS',   _exp('Collect'), { command => 'test'            } ],

        _exp2('Leftovers'),

        [ '@RSRCHBOY/postlude',  _exp('Region'),       { region_name => 'postlude' } ],

        _exp2('SeeAlso'),
        _exp2('SourceGitHub'),
        _exp2('Bugs'),

        [ 'RSRCHBOY::Authors',     _exp('RSRCHBOY::Authors'),     { } ],
        _exp2('Contributors'),
        _exp2('Legal'),

        [ '@RSRCHBOY/List',      _exp('-Transformer'), { transformer => 'List' } ],
        [ '@RSRCHBOY/SingleEncoding', _exp('-SingleEncoding'), {} ],
    );
}

!!42;

__END__

=pod

=encoding UTF-8

=for :stopwords Chris Weyl Bowers Neil Romanov Sergey

=for :stopwords Wishlist flattr flattr'ed gittip gittip'ed

=head1 NAME

Pod::Weaver::PluginBundle::RSRCHBOY - Document your modules like RSRCHBOY does

=head1 VERSION

This document describes version 0.052 of Pod::Weaver::PluginBundle::RSRCHBOY - released November 12, 2014 as part of Dist-Zilla-PluginBundle-RSRCHBOY.

=head1 SYNOPSIS

In weaver.ini:

  [@RSRCHBOY]

or in C<dist.ini>:

  [PodWeaver]
  config_plugin = @RSRCHBOY

=head1 DESCRIPTION

This is the L<Pod::Weaver> config I use for building my
documentation.

=head1 OVERVIEW

This plugin bundle is B<ROUGHLY> equivalent to the following weaver.ini file:

  [@CorePrep]

  [Name]

  [Region / prelude]

  [Generic / SYNOPSIS]
  [Generic / DESCRIPTION]
  [Generic / OVERVIEW]

  [Collect / ATTRIBUTES]
  command = attr

  [Collect / METHODS]
  command = method

  [Collect / FUNCTIONS]
  command = func

  [Collect / TEST_FUNCTIONS]
  command = test

  [Leftovers]

  [Region / postlude]

  [SeeAlso]
  [SourceGitHub]
  [Bugs]
  [Authors]
  [Legal]

  [-Transformer]
  transformer = List

  [-SingleEncoding]

=for Pod::Coverage mvp_bundle_config

=head1 SEE ALSO

Please see those modules/websites for more information related to this module.

=over 4

=item *

L<Dist::Zilla::PluginBundle::RSRCHBOY|Dist::Zilla::PluginBundle::RSRCHBOY>

=back

=head1 SOURCE

The development version is on github at L<http://https://github.com/RsrchBoy/dist-zilla-pluginbundle-rsrchboy>
and may be cloned from L<git://https://github.com/RsrchBoy/dist-zilla-pluginbundle-rsrchboy.git>

=head1 BUGS

Please report any bugs or feature requests on the bugtracker website
https://github.com/RsrchBoy/dist-zilla-pluginbundle-rsrchboy/issues

When submitting a bug or request, please include a test-file or a
patch to an existing test-file that illustrates the bug or desired
feature.

=head1 AUTHOR

Chris Weyl <cweyl@alumni.drew.edu>

=head2 I'm a material boy in a material world

=begin html

<a href="https://www.gittip.com/RsrchBoy/"><img src="https://raw.githubusercontent.com/gittip/www.gittip.com/master/www/assets/%25version/logo.png" /></a>
<a href="http://bit.ly/rsrchboys-wishlist"><img src="http://wps.io/wp-content/uploads/2014/05/amazon_wishlist.resized.png" /></a>
<a href="https://flattr.com/submit/auto?user_id=RsrchBoy&url=https%3A%2F%2Fgithub.com%2FRsrchBoy%2Fdist-zilla-pluginbundle-rsrchboy&title=RsrchBoy's%20CPAN%20Dist-Zilla-PluginBundle-RSRCHBOY&tags=%22RsrchBoy's%20Dist-Zilla-PluginBundle-RSRCHBOY%20in%20the%20CPAN%22"><img src="http://api.flattr.com/button/flattr-badge-large.png" /></a>

=end html

Please note B<I do not expect to be gittip'ed or flattr'ed for this work>,
rather B<it is simply a very pleasant surprise>. I largely create and release
works like this because I need them or I find it enjoyable; however, don't let
that stop you if you feel like it ;)

L<Flattr this|https://flattr.com/submit/auto?user_id=RsrchBoy&url=https%3A%2F%2Fgithub.com%2FRsrchBoy%2Fdist-zilla-pluginbundle-rsrchboy&title=RsrchBoy's%20CPAN%20Dist-Zilla-PluginBundle-RSRCHBOY&tags=%22RsrchBoy's%20Dist-Zilla-PluginBundle-RSRCHBOY%20in%20the%20CPAN%22>,
L<gittip me|https://www.gittip.com/RsrchBoy/>, or indulge my
L<Amazon Wishlist|http://bit.ly/rsrchboys-wishlist>...  If you so desire.

=head1 COPYRIGHT AND LICENSE

This software is Copyright (c) 2013 by Chris Weyl.

This is free software, licensed under:

  The GNU Lesser General Public License, Version 2.1, February 1999

=cut


Makefile.PL
#
# This file is part of Dist-Zilla-PluginBundle-RSRCHBOY
#
# This software is Copyright (c) 2013 by Chris Weyl.
#
# This is free software, licensed under:
#
#   The GNU Lesser General Public License, Version 2.1, February 1999
#

# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.025.
use strict;
use warnings;

use 5.010000;

use ExtUtils::MakeMaker;



my %WriteMakefileArgs = (
  "ABSTRACT" => "Zilla your distributions like RSRCHBOY!",
  "AUTHOR" => "Chris Weyl <cweyl\@alumni.drew.edu>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Dist-Zilla-PluginBundle-RSRCHBOY",
  "EXE_FILES" => [],
  "LICENSE" => "lgpl",
  "MIN_PERL_VERSION" => "5.010000",
  "NAME" => "Dist::Zilla::PluginBundle::RSRCHBOY",
  "PREREQ_PM" => {
    "Archive::Tar::Wrapper" => 0,
    "Config::MVP::Slicer" => "0.302",
    "Dist::Zilla" => 0,
    "Dist::Zilla::App::Command::cover" => 0,
    "Dist::Zilla::Plugin::ArchiveRelease" => 0,
    "Dist::Zilla::Plugin::Authority" => 0,
    "Dist::Zilla::Plugin::CPANFile" => 0,
    "Dist::Zilla::Plugin::CheckChangesHasContent" => 0,
    "Dist::Zilla::Plugin::CheckExtraTests" => 0,
    "Dist::Zilla::Plugin::CheckPrereqsIndexed" => 0,
    "Dist::Zilla::Plugin::CheckSelfDependency" => 0,
    "Dist::Zilla::Plugin::ConfirmRelease" => 0,
    "Dist::Zilla::Plugin::ConsistentVersionTest" => 0,
    "Dist::Zilla::Plugin::ContributorsFromGit" => "0.010",
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
    "Dist::Zilla::Plugin::Git::CommitBuild" => "2.009",
    "Dist::Zilla::Plugin::Git::Describe" => 0,
    "Dist::Zilla::Plugin::Git::NextVersion" => 0,
    "Dist::Zilla::Plugin::Git::Remote::Check" => 0,
    "Dist::Zilla::Plugin::GitHub::Update" => 0,
    "Dist::Zilla::Plugin::GithubMeta" => 0,
    "Dist::Zilla::Plugin::HasVersionTests" => 0,
    "Dist::Zilla::Plugin::InstallRelease" => 0,
    "Dist::Zilla::Plugin::MetaConfig" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaNoIndex" => 0,
    "Dist::Zilla::Plugin::MetaProvides::Package" => 0,
    "Dist::Zilla::Plugin::MetaYAML" => 0,
    "Dist::Zilla::Plugin::MinimumPerl" => 0,
    "Dist::Zilla::Plugin::NoSmartCommentsTests" => 0,
    "Dist::Zilla::Plugin::PodCoverageTests" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::PodWeaver" => "4.000",
    "Dist::Zilla::Plugin::Prepender" => 0,
    "Dist::Zilla::Plugin::PromptIfStale" => 0,
    "Dist::Zilla::Plugin::PruneFiles" => 0,
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
    "Dist::Zilla::Plugin::ReadmeFromPod" => 0,
    "Dist::Zilla::Plugin::ReportVersions::Tiny" => 0,
    "Dist::Zilla::Plugin::Run::AfterMint" => 0,
    "Dist::Zilla::Plugin::RunExtraTests" => 0,
    "Dist::Zilla::Plugin::SchwartzRatio" => 0,
    "Dist::Zilla::Plugin::Signature" => 0,
    "Dist::Zilla::Plugin::SurgicalPkgVersion" => 0,
    "Dist::Zilla::Plugin::TaskWeaver" => 0,
    "Dist::Zilla::Plugin::Test::CheckDeps" => 0,
    "Dist::Zilla::Plugin::Test::Compile" => 0,
    "Dist::Zilla::Plugin::Test::EOL" => 0,
    "Dist::Zilla::Plugin::Test::MinimumVersion" => "2.000005",
    "Dist::Zilla::Plugin::Test::NoTabs" => 0,
    "Dist::Zilla::Plugin::Test::Pod::LinkCheck" => 0,
    "Dist::Zilla::Plugin::Test::PodSpelling" => "2.002001",
    "Dist::Zilla::Plugin::TestRelease" => 0,
    "Dist::Zilla::Plugin::Travis::ConfigForReleaseBranch" => 0,
    "Dist::Zilla::Plugin::Twitter" => 0,
    "Dist::Zilla::Plugin::UploadToCPAN" => 0,
    "Dist::Zilla::PluginBundle::Git" => "1.121770",
    "Dist::Zilla::PluginBundle::Git::CheckFor" => 0,
    "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
    "Dist::Zilla::Role::PluginBundle::Easy" => 0,
    "Dist::Zilla::Role::PluginBundle::PluginRemover" => "0.102",
    "Dist::Zilla::Stash::PAUSE::Encrypted" => "0.003",
    "Git::Raw" => "0.32",
    "Moose" => 0,
    "Moose::Role" => 0,
    "Moose::Util::TypeConstraints" => 0,
    "MooseX::AttributeShortcuts" => "0.023",
    "MooseX::NewDefaults" => 0,
    "MooseX::RelatedClasses" => 0,
    "Path::Class" => 0,
    "Pod::Coverage::TrustPod" => 0,
    "Pod::Elemental::Element::Pod5::Command" => 0,
    "Pod::Elemental::Element::Pod5::Ordinary" => 0,
    "Pod::Elemental::Transformer::List" => 0,
    "Pod::Weaver::Config::Assembler" => 0,
    "Pod::Weaver::Plugin::SingleEncoding" => 0,
    "Pod::Weaver::Plugin::StopWords" => 0,
    "Pod::Weaver::Section::Authors" => 0,
    "Pod::Weaver::Section::CollectWithIntro" => 0,
    "Pod::Weaver::Section::Contributors" => 0,
    "Pod::Weaver::Section::SeeAlso" => 0,
    "Pod::Weaver::Section::SourceGitHub" => 0,
    "Test::MinimumVersion" => 0,
    "Test::NoSmartComments" => 0,
    "Test::Pod" => 0,
    "Test::Pod::Content" => 0,
    "Test::Pod::Coverage" => 0,
    "Test::Pod::LinkCheck" => 0,
    "URI::Escape::XS" => 0,
    "aliased" => 0,
    "autobox::Core" => 0,
    "autodie" => "0.20",
    "namespace::autoclean" => 0,
    "strict" => 0,
    "warnings" => 0
  },
  "TEST_REQUIRES" => {
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::CheckDeps" => "0.010",
    "Test::More" => "0.94"
  },
  "VERSION" => "0.052",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Archive::Tar::Wrapper" => 0,
  "Config::MVP::Slicer" => "0.302",
  "Dist::Zilla" => 0,
  "Dist::Zilla::App::Command::cover" => 0,
  "Dist::Zilla::Plugin::ArchiveRelease" => 0,
  "Dist::Zilla::Plugin::Authority" => 0,
  "Dist::Zilla::Plugin::CPANFile" => 0,
  "Dist::Zilla::Plugin::CheckChangesHasContent" => 0,
  "Dist::Zilla::Plugin::CheckExtraTests" => 0,
  "Dist::Zilla::Plugin::CheckPrereqsIndexed" => 0,
  "Dist::Zilla::Plugin::CheckSelfDependency" => 0,
  "Dist::Zilla::Plugin::ConfirmRelease" => 0,
  "Dist::Zilla::Plugin::ConsistentVersionTest" => 0,
  "Dist::Zilla::Plugin::ContributorsFromGit" => "0.010",
  "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
  "Dist::Zilla::Plugin::Git::CommitBuild" => "2.009",
  "Dist::Zilla::Plugin::Git::Describe" => 0,
  "Dist::Zilla::Plugin::Git::NextVersion" => 0,
  "Dist::Zilla::Plugin::Git::Remote::Check" => 0,
  "Dist::Zilla::Plugin::GitHub::Update" => 0,
  "Dist::Zilla::Plugin::GithubMeta" => 0,
  "Dist::Zilla::Plugin::HasVersionTests" => 0,
  "Dist::Zilla::Plugin::InstallRelease" => 0,
  "Dist::Zilla::Plugin::MetaConfig" => 0,
  "Dist::Zilla::Plugin::MetaJSON" => 0,
  "Dist::Zilla::Plugin::MetaNoIndex" => 0,
  "Dist::Zilla::Plugin::MetaProvides::Package" => 0,
  "Dist::Zilla::Plugin::MetaYAML" => 0,
  "Dist::Zilla::Plugin::MinimumPerl" => 0,
  "Dist::Zilla::Plugin::NoSmartCommentsTests" => 0,
  "Dist::Zilla::Plugin::PodCoverageTests" => 0,
  "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
  "Dist::Zilla::Plugin::PodWeaver" => "4.000",
  "Dist::Zilla::Plugin::Prepender" => 0,
  "Dist::Zilla::Plugin::PromptIfStale" => 0,
  "Dist::Zilla::Plugin::PruneFiles" => 0,
  "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
  "Dist::Zilla::Plugin::ReadmeFromPod" => 0,
  "Dist::Zilla::Plugin::ReportVersions::Tiny" => 0,
  "Dist::Zilla::Plugin::Run::AfterMint" => 0,
  "Dist::Zilla::Plugin::RunExtraTests" => 0,
  "Dist::Zilla::Plugin::SchwartzRatio" => 0,
  "Dist::Zilla::Plugin::Signature" => 0,
  "Dist::Zilla::Plugin::SurgicalPkgVersion" => 0,
  "Dist::Zilla::Plugin::TaskWeaver" => 0,
  "Dist::Zilla::Plugin::Test::CheckDeps" => 0,
  "Dist::Zilla::Plugin::Test::Compile" => 0,
  "Dist::Zilla::Plugin::Test::EOL" => 0,
  "Dist::Zilla::Plugin::Test::MinimumVersion" => "2.000005",
  "Dist::Zilla::Plugin::Test::NoTabs" => 0,
  "Dist::Zilla::Plugin::Test::Pod::LinkCheck" => 0,
  "Dist::Zilla::Plugin::Test::PodSpelling" => "2.002001",
  "Dist::Zilla::Plugin::TestRelease" => 0,
  "Dist::Zilla::Plugin::Travis::ConfigForReleaseBranch" => 0,
  "Dist::Zilla::Plugin::Twitter" => 0,
  "Dist::Zilla::Plugin::UploadToCPAN" => 0,
  "Dist::Zilla::PluginBundle::Git" => "1.121770",
  "Dist::Zilla::PluginBundle::Git::CheckFor" => 0,
  "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
  "Dist::Zilla::Role::PluginBundle::Easy" => 0,
  "Dist::Zilla::Role::PluginBundle::PluginRemover" => "0.102",
  "Dist::Zilla::Stash::PAUSE::Encrypted" => "0.003",
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "Git::Raw" => "0.32",
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Moose" => 0,
  "Moose::Role" => 0,
  "Moose::Util::TypeConstraints" => 0,
  "MooseX::AttributeShortcuts" => "0.023",
  "MooseX::NewDefaults" => 0,
  "MooseX::RelatedClasses" => 0,
  "Path::Class" => 0,
  "Pod::Coverage::TrustPod" => 0,
  "Pod::Elemental::Element::Pod5::Command" => 0,
  "Pod::Elemental::Element::Pod5::Ordinary" => 0,
  "Pod::Elemental::Transformer::List" => 0,
  "Pod::Weaver::Config::Assembler" => 0,
  "Pod::Weaver::Plugin::SingleEncoding" => 0,
  "Pod::Weaver::Plugin::StopWords" => 0,
  "Pod::Weaver::Section::Authors" => 0,
  "Pod::Weaver::Section::CollectWithIntro" => 0,
  "Pod::Weaver::Section::Contributors" => 0,
  "Pod::Weaver::Section::SeeAlso" => 0,
  "Pod::Weaver::Section::SourceGitHub" => 0,
  "Test::CheckDeps" => "0.010",
  "Test::MinimumVersion" => 0,
  "Test::More" => "0.94",
  "Test::NoSmartComments" => 0,
  "Test::Pod" => 0,
  "Test::Pod::Content" => 0,
  "Test::Pod::Coverage" => 0,
  "Test::Pod::LinkCheck" => 0,
  "URI::Escape::XS" => 0,
  "aliased" => 0,
  "autobox::Core" => 0,
  "autodie" => "0.20",
  "namespace::autoclean" => 0,
  "strict" => 0,
  "warnings" => 0
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



